.php_cs.dist

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * Pre-commit hook installation:
+ * vendor/bin/static-review.php hook:install dev/tools/Magento/Tools/StaticReview/pre-commit .git/hooks/pre-commit
+ */
+$finder = PhpCsFixer\Finder::create()
+    ->name('*.phtml')
+    ->exclude('dev/tests/functional/generated')
+    ->exclude('dev/tests/functional/var')
+    ->exclude('dev/tests/functional/vendor')
+    ->exclude('dev/tests/integration/tmp')
+    ->exclude('dev/tests/integration/var')
+    ->exclude('lib/internal/Cm')
```

Comment:
Please exclude `generated` folder also (`var/generation` was moved to `generated`).

Some of these paths are irrelevant already but it will not spoil anything and better to be addressed in separate PR or within this PR processing by Magento folks.